src/cascadia/TerminalApp/TitlebarControl.xaml

```diff
@@ -43,5 +43,12 @@
     <local:MinMaxCloseControl x:Name="MinMaxCloseControl"
                               Grid.Column="2"
                               HorizontalAlignment="Right" />
+
+    <!--  This border needs to be added manually until GH#10320 is fixed  -->

```

Comment:
I would almost reckon that this fixes #10320. From an architectural standpoint, I think it's gonna be **H**ard to make the TabView actually span the whole width, but if this fakes it out, then I'm cool with that as the resolution here.

Reply:
Couldn't the drag bar and caption buttons simply be moved to be inside the TabView footer? Currently they're sitting right next to it, and it seems like it would be easy.

Reply:
Somewhat -- but right now there's an ownership inversion between the titlebar control and the tabbar control. The title bar contains `[tabview content zone, dragbar, caption buttons]`, and they're three distinct areas because when "show tabs in the titlebar" is turned _off_ it doesn't exist and the tabview is kept in the page root.

We'd need to flip around the ownership of the drag bar and caption buttons and pull them out into the titlebar control when "show tabs in the titlebar" was turned on, and otherwise make sure that they are not present when "show tabs in titlebar" is off.